docs/src/development/local/ddev.md

```diff
@@ -36,45 +45,30 @@ If you started with a Drupal template, your repository already had DDEV configur
 Otherwise, you have to answer a couple of questions about what your repository is
 so the correct files are added.
 
-Test the configuration by running the following command:
-
-```bash
-ddev start
-```
-
-Your project should start running, though without any of the data from your Platform.sh environment.
-
 ## 3. Add an API token
 
 To connect DDEV with your Platform.sh account, use a Platform.sh API token.
 
-1. [Create an API token](../../administration/cli/api-tokens.md#2-create-a-platformsh-api-token) in the Console.
-2. Add the token to your DDEV configuration.
-   You can do so globally (easiest for most people):
+1.  [Create an API token](../../administration/cli/api-tokens.md#2-create-a-platformsh-api-token) in the Console.
 
-   ```bash
-   ddev config global --web-environment-add=PLATFORMSH_CLI_TOKEN={{< variable "API_TOKEN" >}}
-   ```
+2.  Add the token to your DDEV configuration.
+    You can do so globally (easiest for most people):
 
-   You can also add the token only to the project:
+    ```bash
+    ddev config global --web-environment-add=PLATFORMSH_CLI_TOKEN={{< variable "API_TOKEN" >}}
+    ```
 
-   ```bash
-   ddev config --web-environment-add=PLATFORMSH_CLI_TOKEN={{< variable "API_TOKEN" >}}
-   ```
+    You can also add the token only to the project:
 
-3. Get DDEV to recognize the token by restarting:
-
-   ```bash
-   ddev restart
-   ```
+    ```bash
+    ddev config --web-environment-add=PLATFORMSH_CLI_TOKEN={{< variable "API_TOKEN" >}}
+    ```
 
 Now you can run `ddev exec platform <command>` from your computer without needing to install the Platform.sh CLI.
```

Comment:
With the exception of the web token creation, all these steps are not needed anymore when using the add-on, since it will ask for the token, the project ID, the main environment, etc.
Of course people may not be using PHP or willing to use the add-on, and we should provide documentation for this use case, but installing the add-on should be the most recommended and advertised way of doing 😉

Reply:
Generally, we try to put as much into commands as possible to make the instructions clearer. So that was the point here -- to help people copy and paste and not have to read interactive prompts. But we can leave it up to the prompts. :shrug:

Reply:
WDYT @rfay ?

Reply:
There's nothing wrong with explicitly setting the PLATFORMSH_CLI_TOKEN in advance IMO, and making it clear to people what they're doing. Side note: If they're using the add-on, `ddev platform` works, don't need `ddev exec platform`. OTOH, `ddev exec platform` works without any caveats.

Reply:
> Side note: If they're using the add-on, ddev platform works, don't need ddev exec platform. OTOH, ddev exec platform works without any caveats.
I'd go with `ddev platform`

Reply:
I'm just afraid that using the add-on may appear complex while it can be more straightforward

Reply:
Keeping it to the essentials is reasonable as you say @lolautruche